tests/test_kubernetes_logs.py

```
@@ -0,0 +1,245 @@
+    assert len(result) == 2
+    assert count == 2
+
+
+def test_filter_logs_include_filter_regex():
+    logs = [
+        _make_log("OOMKilled: memory limit exceeded"),
+        _make_log("info: container started"),
+        _make_log("OOMKilled: again"),
+    ]
+    params = _make_params(filter="OOMKilled")
+    result, count, *_ = filter_logs(logs, params)
```

Ruff (0.15.18)

[warning] Unpacked variable `count` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)